core/scene.go

```diff
@@ -293,6 +293,26 @@ func (sc *Scene) resize(geom math32.Geom2DInt) {
 	sc.NeedsLayout()
 }
 
+// ResizeToContents resizes the scene so it fits the current contents.
+// Only applicable to Desktop systems where windows can be resized.
+// Extra size is added to the amount computed to hold the contents,
+// which is needed in cases with wrapped text elements, which don't
+// always size accurately.
+func (sc *Scene) ResizeToContents(extra image.Point) {
```

Comment:
`ResizeToContent`? Or will people get confused with #1304? Making the user pass `extra` is also not good.

Reply:
made extra optional